keras_hub/src/utils/keras_utils.py

```diff
@@ -104,3 +104,45 @@ def running_on_gpu():
         import torch
 
         return torch.cuda.is_available()
+
+
+def gpu_is_a100():
+    backend = keras.config.backend()
+    if backend == "jax":
+        import jax
+
+        devices = jax.devices()
+        return any(
+            d.platform == "gpu"
+            and "A100" in getattr(d, "device_kind", "").upper()
+            for d in devices
+        )
+    elif backend == "tensorflow":
+        import tensorflow as tf
```

Comment:
Split this function in two. A function that returns a normalized GPU name regardless of backend. And this function which uses the normalize name to compare with a denylist.

Reply:
done